etc/notebooks/sentiment_boilerplate_demo/sentiment_boilerplate_demo.ipynb

```diff
@@ -0,0 +1,797 @@
+    "\n",
+    "\n",
+    "For example, in the call center scenraio a proper return value might be:\n",
+    "\n",
+    "```\n",
+    "[\n",
+    "    ['Call Wait Time', 10, 20],\n",
+    "    ['Manager Involvement', 25, 5]\n",
+    "]\n",
+    "```\n",
+    "\n",
+    "In this example we define positive `Call Wait Time` when a customer waited for less than 10 minutes. Therefore,  negative `Call Wait Time` is when a custimer waited 10 minutes or more for their call to be handled. Positive `Manager Involvment` is defined as not having to invovle a manager during the call, whereas negative `Manager Involvment` involves a manager."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
+    "urth": {
+     "dashboard": {}
+    }
+   },
+   "outputs": [],
+   "source": [
+    "def get_sentiment_counts_during(start_date, end_date):\n",
+    "    '''Returns sentiment counts (positive and negative) for indiviudal metrics. \n",
+    "    '''    \n",
+    "    return [\n",
+    "        ['Metric A', random.randint(50,60), random.randint(0,10)],\n",
+    "        ['Metric B', random.randint(20,30), random.randint(40,50)],\n",
+    "        ['Metric C', random.randint(70,80), random.randint(10,20)]\n",
+    "    ]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "This constant allows you to change the length of the current sentiment window. This affects the outcome of [`get_dashboard_data`](#get_dashboard_data) which will affect the total and individual sentiment count charts."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
+    "urth": {
+     "dashboard": {}
+    }
+   },
+   "outputs": [],
+   "source": [
+    "CURRENT_WINDOW_SIZE_IN_DAYS = 7"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "This constant controls the number of data points returned by [`get_historical_sentiment_data`](#get_historical_sentiment_data). You can increase or decrease this number to respectively get more or less data points."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true,
+    "urth": {
+     "dashboard": {}
+    }
+   },
+   "outputs": [],
+   "source": [
+    "HISTORICAL_WINDOW_SLICES = 30"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "##Boilerplate"
```

Comment:
I'm assuming I don't have to change anything below here. That's worth calling out in a big bold letters. In fact, the table of contents at the top maybe should become a ordered sequence of things you should do. Something like:

1. Run this notebook to understand what it gives you
2. Read the Extension Point section
3. Implement your code there
4. Run it again to see your own data